contracts/Strategy.sol

```diff
@@ -119,24 +176,30 @@ contract Strategy is BaseStrategy {
 
         uint256 beforeWant = balanceOfWant();
 
-        collectTradingFees();
+        _collectTradingFees();
         // claim beets
-        claimRewards();
-        // consolidate % to stake and unstake
-        consolidate();
-        // sell the % not staking
-        sellRewards();
+        _claimRewards();
+        // this would allow finer control over harvesting to get credits in without selling
+        if (toggles.doSellRewards) {
+            _sellRewards();
+        }
```

Comment:
Probably should have noted this in a previous review, but doesn't `liquidatePosition` and `_collectTradingFees` trading fees duplicate effort? Both unstake and sell bpts.

Reply:
`_collectTradingFees` forces an exit while `liquidatePosition` could potentially just use loose wants if there are any. Using loose wants would cause an accounting error if `debtOutstanding > 0`. It wouldn't actually exit the extra trading fees